public/scripts/templates/welcome.html

```diff
@@ -11,8 +11,18 @@ <h3 data-i18n="How to start chatting?">How to start chatting?</h3>
         <span data-i18n="Click ">Click </span><code><i class="fa-solid fa-plug"></i></code><span data-i18n="and select a"> and select a </span><a href="https://docs.sillytavern.app/usage/api-connections/" target="_blank" data-i18n="Chat API">Chat API</a>.</span>
     </li>
     <li>
-        <span data-i18n="Click ">Click </span><code><i class="fa-solid fa-address-card"></i></code><span data-i18n="and pick a character"> and pick a character</span>
+        <span data-i18n="Click ">Click </span><code><i class="fa-solid fa-address-card"></i></code><span data-i18n="and pick a character."> and pick a character.</span>
     </li>
+    <em>
+        <ul>
+            <li>
+                <span data-i18n="You can browse a list of bundled characters by clicking ">You can browse a list of bundled characters by clicking </span><code><i class="fa-solid fa-cubes"></i></code><span data-i18n="followed by Download & Extensions Assets"> followed by <b>Download & Extensions Assets.</b></span>
```

Comment:
I can't say I like the way this instruction is worded as it goes out of line with otherwise minimalistic welcome text guidance. It's a very direct clicking guide rather than giving a direction and allowing one to explore. I think just the first part would be enough to nudge toward the path they should take.

Also, I'd like to request a slightly different styling. Put it out of the nested list to reduce the left margin.

Reply:
You can find bundled characters in the "Download Extensions and Assets" menu under the "blocks" icon.
Something like this.